core-libs/setup/ssr/error-handling/server-error-response-factory/unknown-server-error-response-factory.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Priority } from '@spartacus/core';
+import { UnknownServerErrorResponseFactory } from './unknown-server-error-response-factory';
+import { TestBed } from '@angular/core/testing';
+
+describe('UnknonnServerErrorResponse', () => {
```

Comment:
```suggestion
describe('UnknownServerErrorResponse', () => {
```

PS. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker for the win